src/_guides/language/language-tour.md

```diff
@@ -3713,6 +3586,16 @@ Future checkVersion() [!async!] {
 }
 {% endprettify %}
 
+<aside class="alert alert-info" markdown="1">
+**Note:**
+Although an async function might perform time-consuming operations,
+it doesn't wait for those operations.
+Instead, the async function executes only until it encounters
+its first `await` expression.
+Then it returns a Future object,
```

Comment:
[Style nit] Future --> `Future`

(I'm not sure that we are consistent with this convention. E.g., one of the changes you made early in this page was to use \<code> style for `String`, `int`, etc. but I don't think that we are systematic about it.)

Reply:
We aren't consistent. At first I was trying to minimize the use of code font, especially in words that are already capitalized (this was the style at NeXT, but code font keeps creeping in.